src/test/resources/standalone/config/build-config.json

```diff
@@ -30,6 +30,13 @@
     "ignoreFilePattern" : "(temp|tmp)",
     "ferryIdsNotAllowedForBicycle" : ["RUT:B107", "RUT:B209"]
   },
+  "gtfsDefaults" : {
+    "stationTransferPreference" : "allowed",
+    "removeRepeatedStops" : true,
+    "discardMinTransferTimes": false,
+    "blockBasedInterlining": true,
+    "maxInterlineDistance": 200
+  },
```

Comment:
It came to me as a surprise that these values defined here are actually used in the documentation as defaults in some places. I would have expected these just to be examples.

Reply:
I'm quite surprised by that as well. I think it's worth investigating.

Reply:
I made a PR to fix this, can you take a look @optionsome ?

Reply:
#4640